.circleci/config.yml

```diff
@@ -139,15 +139,15 @@ commands:
         steps:
             - restore_cache:
                 keys:
-                    - 4-pods-{{ arch }}-{{ checksum "native/ios/Podfile.lock" }}
-                    - 4-pods-{{ arch }}-
+                    - 5-pods-{{ arch }}-{{ checksum "native/ios/Podfile.lock" }}
+                    - 5-pods-{{ arch }}-
```

Comment:
What is the prefix actually for? 😅

Reply:
To force completely reinstalling all pods. The key is different then and no old cache will be used.